pkg/cloud/ibmcloud/clients/machines.go

```diff
@@ -104,3 +114,25 @@ func (gs *GuestService) GuestList() ([]datatypes.Virtual_Guest, error) {
 	}
 	return guests, nil
 }
+
+func getSshKey(sess *session.Session, name string) int {
+	id := 0
+
+	service := services.GetAccountService(sess)
+	keys, err := service.GetSshKeys()
+	if err != nil {
+		log.Printf("Error retrieving ssh keys from Account: %s\n", err)
+		return id
+	}
+
+	for _, key := range keys {
+		if *key.Label == name {
+			id = *key.Id
+			log.Printf("Get SSH key for %q with value %d\n", *key.Label, *key.Id)
+			break
+		}
```

Comment:
do we want to add a error case here to know the key is not there? or you will give some error like 'id 0' not found which will be weird?